build.yaml

```diff
@@ -2896,3 +2858,88 @@ steps:
     - deploy_batch
     - deploy_shuffler
     - deploy_address
+ - kind: runImage
+   name: test_batch_invariants
+   image:
+     valueFrom: batch_image.image
+   script: |
+     python3 -m pytest --log-date-format="%Y-%m-%dT%H:%M:%S" --log-format="%(asctime)s %(levelname)s %(name)s %(filename)s:%(lineno)d:%(funcName)s %(message)s" --log-cli-level=INFO -s -vv --instafail -k "test_invariants" /io/test/
+   inputs:
+    - from: /repo/batch/test
+      to: /io/
+   timeout: 300
+   secrets:
+    - name: gce-deploy-config
+      namespace:
+        valueFrom: default_ns.name
+      mountPath: /deploy-config
+    - name: test-dev-tokens
+      namespace:
+        valueFrom: batch_pods_ns.name
+      mountPath: /user-tokens
+    - name: ssl-config-batch-tests
+      namespace:
+        valueFrom: default_ns.name
+      mountPath: /ssl-config
+   scopes:
+    - test
```

Comment:
Actually, we probably could run this in deploy mode. The errors will always be None though.